bindata/network/ovn-kubernetes/common/ipsec.yaml

```diff
@@ -251,24 +268,50 @@ spec:
         beta.kubernetes.io/os: "linux"
       terminationGracePeriodSeconds: 10
       volumes:
-      - name: host-var-log-ovs
-        hostPath:
+      - hostPath:
           path: /var/log/openvswitch
           type: DirectoryOrCreate
-      - name: host-var-run-ovs
-        hostPath:
+        name: host-var-log-ovs
+      - hostPath:
           path: /var/run/openvswitch
           type: DirectoryOrCreate
-      - name: signer-ca
-        configMap:
+        name: host-var-run-ovs
+      - configMap:
+          defaultMode: 420
           name: signer-ca
-      - name: etc-openvswitch
-        hostPath:
+        name: signer-ca
```

Comment:
It doesn't look like there is an impact on cert-rotation, but asking anyway for [SDN-4039](https://issues.redhat.com/browse/SDN-4039)

Reply:
the impact might be related to the fact that the NSS DB is now persistent, so better remove old cert&keys otherwise eventually it'll explode.